lib/rule-tester/rule-tester.js

```diff
@@ -768,6 +772,9 @@ class RuleTester {
 		assertRule(rule, ruleName);
 		assertTest(test, ruleName);
 
+		const { requireMessage = false, requireLocation = false } =
```

Comment:
In e2516713bc9ae62117da3f490d9cb6a9676f44fe, the assertions unrelated to execution have been extracted into separate functions. Could you move these newly added checks into `assertInvalidTestCase()`?

https://github.com/eslint/eslint/blob/a2ee188ea7a38a0c6155f3d39e2b00e1d0f36e14/lib/rule-tester/rule-tester.js#L560

Reply:
I can move the requireMessage assertions to the other method, but requireLocation requires the actual execution.
For `requireMessage` I have to rebuild some of the structure for `testInvalidTemplate`.
Is this what you expect?